### PR TITLE
Add recitations to seed data.

### PR DESCRIPTION
**Pull request by spark008**

### CODE REVIEW

**Comment by pwnall** [comment]
Discussed in person. Thank you very much for doing this!

**Comment by spark008** [comment]
The pleasure is mine :grin: I've made some changes - what do you think?

File: db/seeds.rb [hunk 1]
```diff
@@ -108,6 +108,68 @@
...
+
+time_slot_ids = time_slot_data.map do |data|
+  time_slot = TimeSlot.new data
+  time_slot.course = course
+  time_slot.save!
+  time_slot.id
+end
+
+recitation_data = [
+  { course: course, leader: staff[0], serial: 10, location: 'Room 0',
```
Comment (pwnall) [inline db/seeds.rb]: I think you can remove course, because it's not variable

File: db/seeds.rb [hunk 2]
```diff
@@ -108,6 +108,68 @@
 
 puts 'Staff created'
 
+# Recitations.
+
+time_slot_data = [
+  { day: 0, starts_at: 1000, ends_at: 1200 },  # All students have conflict here
```
Comment (pwnall) [inline db/seeds.rb]: `conflict` -> `conflicts`?

File: db/seeds.rb [hunk 3]
```diff
@@ -108,6 +108,68 @@
...
+time_slot_ids = time_slot_data.map do |data|
+  time_slot = TimeSlot.new data
+  time_slot.course = course
+  time_slot.save!
+  time_slot.id
+end
+
+recitation_data = [
+  { course: course, leader: staff[0], serial: 10, location: 'Room 0',
+    time_slot_ids: [0, 3] },
```
Comment (pwnall) [inline db/seeds.rb]: If things still don't fit on one line after removing the course key, you can consider using `slot_ids` here, and using `data[:time_slot_ids]` = data.delete(:slot_ids).map ...` below.

File: db/seeds.rb [hunk 4]
```diff
@@ -23,7 +23,7 @@
 course.update! number: '1.337', title: 'Intro to Pwnage',
     email: '1.337-staff@mit.edu', heap_appid: '2171986850',
     email_on_role_requests: true, has_recitations: false, has_surveys: false,
```
Comment (pwnall) [inline db/seeds.rb]: `has_recitations` should probably become true

File: db/seeds/names.txt [hunk 5]
```diff
@@ -1,4 +1,3 @@
-0.   John Doe
```
Comment (pwnall) [inline db/seeds/names.txt]: Oh noes, John will be missed.

**Comment by pwnall** [comment]
LGTM after small changes.

**Pull request merged**